os/mm/kmm_heap/kmm_calloc.c

```diff
@@ -146,10 +147,9 @@ void *kmm_calloc_at(int heap_index, size_t n, size_t elem_size)
 
 FAR void *kmm_calloc(size_t n, size_t elem_size)
 {
+	size_t caller_retaddr;
```

Comment:
Need to change line 157 according to this

Reply:
Modified. Thanks.